generators/entity-client/templates/react/src/main/webapp/app/entities/_entity.reducer.ts

```diff
@@ -154,6 +171,9 @@ _%>
 };
 
 const apiUrl = <% if (applicationType === 'gateway' && locals.microserviceName) { %>'/<%= microserviceName.toLowerCase() %>/<% } else if (authenticationType === 'uaa') { %>'<% } else { %>SERVER_API_URL + '<% } %>/api/<%= entityApiUrl %>';
+<%_ if (searchEngine === 'elasticsearch') { _%>
+const apiSearchUrl = <% if (applicationType === 'gateway' && locals.microserviceName) { %>'/<%= microserviceName.toLowerCase() %>/<% } else if (authenticationType === 'uaa') { %>'<% } else { %>SERVER_API_URL + '<% } %>/api/_search/<%= entityApiUrl %>';
```

Comment:
we need to simplify this part following what is done in angular recently to use a single variable instead of all these checks

Reply:
we have the same problem on _entity.service.ts (angular) ...

Reply:
oh ok, i think the improvement was done only on the non entity services like this `private resourceUrl = SERVER_API_URL + '<%- apiUaaPath %>api/users';` anyway then lets leave it for now

Reply:
ok